src/commons/scheduler.py

```diff
@@ -84,13 +93,15 @@ async def schedule_sessions(test_plan: str, test_plan_value: dict, common_params
                                         start_time=each["start_time"].total_seconds(), **params))
         else:
             raise NotImplementedError(f"Tool is not supported: {each['tool']}")
-    done, pending = await asyncio.wait(tasks, return_when=asyncio.FIRST_COMPLETED)
-    LOGGER.critical("Terminating process & pending task: %s", process_name)
-    for task in pending:
-        task.cancel()
-    LOGGER.critical(done)
-    for task in done:
-        task.result()
+    done, pending = await asyncio.wait(tasks, return_when=asyncio.FIRST_EXCEPTION)
```

Comment:
As discussed, process termination for sequential run is not handled in this PR.

Reply:
This will be taken care in RVF task.